packages/expect/src/__tests__/matchers.test.js

```diff
@@ -198,6 +216,8 @@ describe('.toBe()', () => {
     jestExpect(null).toBe(null);
     jestExpect(undefined).toBe(undefined);
     jestExpect(NaN).toBe(NaN);
+    jestExpect(asBigInt(1)).not.toBe(2);
```

Comment:
It looks like these are only occurrences of `asBigInt` so is it possible to put 2 `jestExpect` calls into `if` block and call `BigInt` directly, following pattern in rest of file?

Reply:
Yeah I originally had this instead of the if blocks,then changed course when realized it wasn't going to pass CI tests with different snapshots. Must of forgotten about these. Thanks